### ISSUE TITLE
Bug report: make prep leaves behind an untracked file

### ISSUE THREAD

**Issue by lcn2** [ioccc-src/mkiocccentry#518]
## Describe the bug

The file `jparse/lex.jparse_.c` is created during the _make_action 24 bug_report-txl_ phase of `make prep`.
This file is NOT removed by `make clobber`, nor it is ignored by git.

## To Reproduce

Steps to reproduce the behavior (adjust steps as needed):

0. make clobber
1. git status
2. make prep
3. git status

## Expected behavior

After steps 1 and 3, no untracked files are reported.

## Relevant Output

After step 3, `git status` reports:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	jparse/lex.jparse_.c
```

## bug report attachment

We request that you **PLEASE** run the command:

```sh
./bug_report.sh
```

and attach the bug report file that `./bug_report.sh` generated.
The bug report file will be a filename in the same directory with a
filename of the form: `bug-report.YYYYMMDD.HHMMSS.txt` and it will tell you the
name as well.

vvvvvvvvvvvvvvvvvvvvvvvvv --- Drag the file into the area below, please!

[bug-report.20230521.185045.txt](https://github.com/ioccc-src/mkiocccentry/files/11526185/bug-report.20230521.185045.txt)

^^^^^^^^^^^^^^^^^^^^^^^^^ --- Drag the file into the area above, please!

## Please provide the following information:

 - Did you attach the above requested bug report file?  [yes, no & why not]
 - What is the airspeed Velocity of an Unladen Swallow? [n/a, :-), X m/s, unknown, etc.]

## Additional context

This problem was observed under macOS 13.4.

Doing a little debugging with the `test_ioccc/prep.sh` script shows that the `jparse/lex.jparse_.c` file
is created during the _make_action 24 bug_report-txl -L ...` action phase.

Why is the `jparse/lex.jparse_.c` file created?  Is the creation of the file a bug or mis-feature or an actual feature?
If it is OK for the file to be created, then why doesn't `make clobber` remove it?
If it is OK for the file to be created, AND `make clobber` is changer to remove it, should it be added to `.gitignore`?

## Thank you, sorry (tm Canada :-) ), next steps

We are sorry (tm Canada :-)) that you seem to have encountered an problem or issue that prompted
you to file this issue.  We **DO** appreciate your taking the time to inform us.

**Please be patent!**  We work as volunteers and have a number of other pressing matters.
While we are often **very busy** we **WILL** look at your issue.

After you click the ((Submit new issue)) button, we suggest you bookmark the new issue page.
We might have additional questions for you, and possibly a request for you to fetch new
code for you to try.  If so, we will add additional comments to the issue.

In some cases we might need to delay fixing the problem because we are working on a larger
more general issue that will eventually and hopefully resolve your issue.

Once you are ready, *please* recheck:

* The issue title at the top

```
NOTE: Does it start with "bug report: "???
```

* The text above

```
NOTE: Being both both **clear** and **concise** helps us.
```

* The attached bug report

```
You should see a line of the form:
\[Untitled.txt\]\(https://github.com/ioccc-src/mkiocccentry/files/some_digits/bug-report.YYYYMMDD.HHMMSS.txt\)
```

Once you have rechecked the above, please click the *green* ((Submit new issue)) button vvvvvvvvvvvvvvvvvvvvvvvvv

**Comment by xexyl** [comment]
I had another somewhat important commit to do .. this issue now has been resolved but testing (make clobber tested; testing other but should be fine: it's a simple matter of updating the `rm -f` command at the end of the `bug_report.sh` script in clean up). I'll update `CHANGES.md` as well before doing a new pull request.

**Comment by xexyl** [comment]
With commit https://github.com/ioccc-src/mkiocccentry/pull/520/commits/ad9d8550450f944672e0cb81b2896049afd80a0b this <strike>bogus</strike> buggy file is now removed in both `make clobber` and in the `bug_report.sh` script itself.

Thanks for the report!

**Comment by xexyl** [comment]
> 
> Thanks for the report!

This seems like a good opportunity, incidentally, to bring up again [GitHub issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms). 

The image a bit below that shows what it looks like and it seems vastly superior to what we have. I can maybe look at doing this in a test repo to make sure everything appears okay but not right now. What do you think of this idea? i'm sure you do agree with it but still think it's good to bring it up either way.

**Comment by xexyl** [comment]
Re issue forms: I've created a test repo to play round with this feature [here](https://github.com/xexyl/github-features).

However I am too tired right now to do the actual playing so I'll have to return to this later .. maybe today or maybe not .. we shall see! I hopefully do not forget this entirely but maybe with replies to this issue (which I fully expect) I can be reminded of this ... as I'm even too tired right now to make a reminder and too lazy to use Siri atm too :-)
